tbapy/main.py

```diff
@@ -1,109 +1,343 @@
-import requests
+from requests import get
+from .models import *
```

Comment:
This is also in `__init__.py`. I'm not too familiar with putting stuff there, but is this needed?